data/en/stocks_0010.json

```diff
@@ -123,8 +123,8 @@
                                         {
                                             "list": [
                                                 "Products that you own in intermediate stages of completion (even if not held by you)",
-                                                "Long term business contract balances in line with UK Generally Accepted Accountancy Practice or International GAAP",
-                                                "All stocks owned; whether held by you or in transit in the UK or abroad",
+                                                "Long term business contract balances in line with UK Generally Accepted Accountancy Practice (GAAP) or International GAAP",
```

Comment:
There is no definition of GAAP but it is used. I'm no writing expert but this looks weird

Reply:
Generally Accepted Accountancy Practice?

Reply:
Yeah I know it is but this change removes the definition. That's what I think is weird.

I only know what GAAP means by either: guessing from the sentence context or, by knowing it up front.

Reply:
You can ignore this as this is content. I just think its odd

Reply:
Doesn't this change add the definition?